android/src/main/kotlin/live/hms/hmssdk_flutter/HmssdkFlutterPlugin.kt

```diff
@@ -934,14 +926,6 @@ class HmssdkFlutterPlugin :
         }
 
         override fun onPeerUpdate(type: HMSPeerUpdate, peer: HMSPeer) {
-            if (type == HMSPeerUpdate.AUDIO_TOGGLED || type == HMSPeerUpdate.VIDEO_TOGGLED ||
```

Comment:
`type == HMSPeerUpdate.BECAME_DOMINANT_SPEAKER || type == HMSPeerUpdate.NO_DOMINANT_SPEAKER`

these checks are not to be removed